tests/disaster-recovery/regional-dr/test_failover.py

```diff
@@ -82,10 +133,15 @@ def test_failover(
                 f"Waiting for {wait_time} minutes before starting nodes of previous primary cluster"
             )
             sleep(wait_time * 60)
+            config.switch_ctx(index=1)
```

Comment:
instead of this use the specific function to switch to the particular cluster

Reply:
done